app/controllers/users/passwords_controller.rb

```diff
@@ -53,5 +63,9 @@ def handle_invalid_password
       end
       render :edit
     end
+
+    def store_location(url)
```

Comment:
why make this a method? rubocop ABC complexity?

Reply:
it was, but with the other changes it isn't necessary anymore, so I've removed the method